### DISCUSSION TITLE
tag LuogoFiliazione

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#953] [Supporto servizi cooperativi] [answered]
Buongiorno,
ho trovato un'incoerenza nella documentazione. Direi che la parte del modello multilinguismo sia errato visto che la tabella 19 non esiste assolutamente e che nel modello italiano ci sia una semplice descrizione.

italiano
![image](https://github.com/italia/ansc/assets/127182190/ef54030b-9c86-4449-bdb4-cea78c05e594)

multilingua
![image](https://github.com/italia/ansc/assets/127182190/b502d8c2-6929-41d1-ad76-cc45bbb5368a)

**Comment by amusarra** [comment, marked as answer]
Salve @annamaraspin 
Abbiamo verificato che c'è un refuso.
Trasformiamo la discussione in issue.

Cordiali Saluti.
Supporto ANSC.
